SpatialGDK/Source/Public/Schema/UnrealObjectRef.h

```diff
@@ -63,22 +60,20 @@ struct UnrealObjectRef
 	improbable::TSchemaOption<UnrealObjectRef> Outer;
 };
 
-}
-
 // This template specialization informs THasGetTypeHash
-// that there exists a valid GetTypeHash function for improbable::UnrealObjectRef.
+// that there exists a valid GetTypeHash function for UnrealObjectRef.
 // This is required as THasGetTypeHash fails to resolve GetTypeHash functions referencing 
 // classes inside namespaces.
 template <>
-struct THasGetTypeHash<improbable::UnrealObjectRef>
+struct THasGetTypeHash<UnrealObjectRef>
```

Comment:
You can remove this struct as well since it was only needed because of the namespace.